handlers/admin/data-contributions.js

```diff
@@ -16,23 +20,30 @@ function editDataContribution(request, reply) {
       request.yar.flash('success', 'Data Contribution updated successfully.');
       return reply.redirect('/admin/data-contributions');
     })
-    .catch(() => {
-      // FIXME: Log error
+    .catch((error) => {
+      console.error(error.detail); // eslint-disable-line no-console
```

Comment:
I'm not sure what's in `error.detail`, but we log as `console.error(error)` in the API and it returns the traceback. Feel free to keep whatever you prefer, just saying in the unlikely case you haven't tried it as well.